js/components/frame.js

```diff
@@ -965,6 +965,11 @@ class Frame extends ImmutableComponent {
         }))
       }
     })
+    this.webview.addEventListener('did-get-response-details', (details) => {
+      if (getSetting(settings.PAYMENTS_ENABLED) && details.originalURL === this.props.location) {
```

Comment:
I thought I commented on this. I think we should always send it. The event store will be used for more than just ledger